requirements.txt

```diff
@@ -18,3 +18,4 @@ gsutil
 cvss
 packaging<22.0
 importlib_resources; python_version < "3.9"
+backports.entry-points-selectable
```

Comment:
We don't need this dependency. Instead just require `importlib_metadata >= 3.6` above.
With this additional constraint imports in other files will stay the same (as in the main now) but usage should change (as you have in this PR).

Reply:
> The “selectable” entry points were introduced in importlib_metadata 3.6 and Python 3.10.

https://docs.python.org/3/library/importlib.metadata.html

So you will also need to require `importlib_metadata` on Pythons less than 3.10 (right now we require it on Pythons less than 3.8). Still better than requiring new dependency on all Pythons.